docs/src/pages/components/typeahead.mdx

```diff
@@ -0,0 +1,148 @@
+---
+name: Typeahead
+route: /components/viewtoggle
+---
+
+import Typeahead from '@pluralsight/ps-design-system-typeahead'
+
+import { Types, Usage } from '../../components/component'
+
+# Typeahead
+
+## In-app example
+
+```typescript startExpanded=false
+import Typeahead from '@pluralsight/ps-design-system-typeahead'
+import React from 'react'
+
+const Example: React.FC = props => {
+  const options = ['Beginner', 'Intermediate', 'Advanced']
+
+  return (
+    <Typeahead label="Level" placeholder="Select a Level">
+      {options.map((option, key) => (
+        <Typeahead.Suggestion key={key}>{option}</Typeahead.Suggestion>
+      ))}
+    </Typeahead>
+  )
+}
+
+export default Example
+```
+
+## Appearance
+
+For a more subtle look, use `subtle` appearance. So subtle.
+
+```typescript startExpanded=false
+import Typeahead from '@pluralsight/ps-design-system-typeahead'
+import React from 'react'
+
+const Example: React.FC = props => {
+  return (
+    <Typeahead
+      appearance={Typeahead.appearances.subtle}
+      placeholder="Search"
+    />
+  )
+}
+
+export default Example
+```
+
+## Size
+
+The small typeahead is ideal for usage within table rows otherwise use the default, medium size typeahead, in forms for example.
+
+```typescript startExpanded=false
+import Typeahead from '@pluralsight/ps-design-system-typeahead'
+import React from 'react'
+
+const Example: React.FC = props => {
+  return (
+    <div className="example-grid--col-2">
+      <Typeahead placeholder="medium typeahead" />
+      <Typeahead placeholder="small typeahead" size={Typeahead.sizes.small} />
+    </div>
+  )
+}
+
+export default Example
+```
+
+## Usage & types
+
+<Usage
+  install="npm install @pluralsight/ps-design-system-typeahead"
+  import="import VerticalTabs from '@pluralsight/ps-design-system-typeahead'"
+  packageName="typeahead"
+/>
+
+### Typeahead
+
+<Types.Table>
```

Comment:
Looks like someone got prettier to deal with mdx files. 👍   I need to do that sometime here.